From 8f9749afb81f8068dbe33bc671797f37023b1f38 Mon Sep 17 00:00:00 2001
From: Andrei Botila <andrei.botila@nxp.com>
Date: Tue, 28 Jun 2022 12:23:00 +0300
Subject: [PATCH 1/6] fdts: s32: change the clock-name from ts to ptp_ref for
 gmac

GMAC clock-naming for TS has been changed in Linux from TS to PTP_REF.
Make the necessary change since we have code in U-Boot that searches for
"ptp_ref" clocking to initialize.

Issue: ALB-8981
Upstream-Status: Pending 

Signed-off-by: Andrei Botila <andrei.botila@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 fdts/s32g.dtsi      | 2 +-
 fdts/s32r45-evb.dts | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/fdts/s32g.dtsi b/fdts/s32g.dtsi
index 9fa29a86c..be36fcadc 100644
--- a/fdts/s32g.dtsi
+++ b/fdts/s32g.dtsi
@@ -278,7 +278,7 @@
 		 <&clks S32GEN1_SCMI_CLK_GMAC0_RX_RGMII>,
 		 <&clks S32GEN1_SCMI_CLK_GMAC0_TX_RGMII>,
 		 <&clks S32GEN1_SCMI_CLK_GMAC0_AXI>;
-	clock-names = "ts", "rx_sgmii", "tx_sgmii",
+	clock-names = "ptp_ref", "rx_sgmii", "tx_sgmii",
 		      "rx_rgmii", "tx_rgmii", "axi";
 	pinctrl-0 = <&pinctrl0_gmac0 &pinctrl0_gmac0_mdio>;
 	pinctrl-1 = <&pinctrl0_gmac0_mdio>;
diff --git a/fdts/s32r45-evb.dts b/fdts/s32r45-evb.dts
index 776e7062e..69cd73864 100644
--- a/fdts/s32r45-evb.dts
+++ b/fdts/s32r45-evb.dts
@@ -52,7 +52,7 @@
 		 <&clks S32GEN1_SCMI_CLK_GMAC0_RX_RGMII>,
 		 <&clks S32GEN1_SCMI_CLK_GMAC0_TX_RGMII>,
 		 <&clks S32GEN1_SCMI_CLK_GMAC0_AXI>;
-	clock-names = "ts", "rx_sgmii", "tx_sgmii",
+	clock-names = "ptp_ref", "rx_sgmii", "tx_sgmii",
 		      "rx_rgmii", "tx_rgmii", "axi";
 	pinctrl-0 = <&pinctrl0_gmac0 &pinctrl0_gmac0_mdio>;
 	pinctrl-1 = <&pinctrl0_gmac0_mdio>;
-- 
2.17.1

